src/components/Tooltip/TooltipWithPortal.tsx

```diff
@@ -1,36 +1,58 @@
-import { useCallback, useRef, useState } from "react";
+import React, { useCallback, useRef, useState } from "react";
 import cx from "classnames";
 
 import "./Tooltip.css";
 import { IS_TOUCH } from "config/env";
 import Portal from "../Common/Portal";
+import Wrapper from "./Wrapper";
 
 const OPEN_DELAY = 0;
 const CLOSE_DELAY = 100;
 
-export default function TooltipWithPortal(props) {
+type Props = {
+  handle: React.ReactNode;
+  renderContent: () => React.ReactNode;
+  position?: string;
+  trigger?: string;
+  className?: string;
+  disableHandleStyle?: boolean;
+  handleClassName?: string;
+  onDisabled?: boolean;
```

Comment:
1. mm, onDisabled sounds like an event handler callback

Not sure properly naming here, maybe isHandlerDisabled/disablePointerEvents?

2. I think comment should be here instead of Wrapper
// For onMouseLeave to work on disabled button https://github.com/react-component/tooltip/issues/18#issuecomment-411476678

3. I'm not sure we need a Wrapper component with so little functionality, Wrapper sounds like something global, not just a specific rare case handler. Maybe leaving a conditional div or className event is enough.

Reply:
Sure!